packages/core/src/associations/has-one.ts

```diff
@@ -97,14 +98,12 @@ export class HasOne<
     }
 
     if ('keyType' in options) {
-      throw new TypeError('Option "keyType" has been removed from the BelongsTo\'s options. Set "foreignKey.type" instead.');
+      throw new TypeError(`Option "keyType" has been removed from the BelongsTo's options. Set "foreignKey.type" instead.`);
     }
 
-    // TODO: throw is source model has a composite primary key.
```

Comment:
This is not something we want anymore?

Reply:
It is, but belongsTo is the association that handles creating the fk so this comment is redundant